fluss-server/src/main/java/com/alibaba/fluss/server/kv/KvTablet.java

```diff
@@ -267,12 +271,27 @@ public LogAppendInfo putAsLeader(
                                 KvRecordReadContext.createReadContext(kvFormat, fieldTypes);
                         ValueDecoder valueDecoder =
                                 new ValueDecoder(readContext.getRowDecoder(schemaId));
-
+                        RowMergeEngine rowMergeEngine = getRowMergeEngine(schema);
```

Comment:
This is a heavy operation, the row merger should be initialized when kv tablet is setup. And a dynamic `PartialUpdateRowMerger` will be used if the `put` is a partial update.